source/_integrations/modbus.markdown

```diff
@@ -1095,70 +1378,170 @@ modbus:
       required: false
       type: map
       keys:
         address:
-          description: Address to read from.
+          description: "Address to read from."
           required: false
-          default: write address
+          default: "Same as `write address`"
           type: integer
         delay:
-          description: delay between write and verify.
+          description: "Delay between write and verify."
           required: false
           default: 0
           type: integer
         input_type:
-          description: type of address (holding/coil/discrete/input or holdings/coils for array call)
+          description: Type of address.
           required: false
-          default: write_type
-          type: integer
+          default: same as `write_type`
+          type: map
+          keys:
+            coil:
+              description: "Coil (1bit relay)."
+            discrete:
+              description: "Discret input (1bit relay)."
+            holding:
+              description: "Holding register."
+            input:
+              description: "Input register."
         state_on:
-          description: value when switch is on.
+          description: "Value when switch is on."
           required: false
-          default: same as command_on
+          default: "Same as `command_on`"
           type: integer
         state_off:
-          description: value when switch is off.
+          description: "Value when switch is off."
           required: false
-          default: same as command_off
+          default: "Same as `command_off`"
           type: integer
-    unique_id:
-      description: An ID that uniquely identifies this sensor. If two sensors have the same unique ID, Home Assistant will raise an exception.
-      required: false
-      type: string
+
 {% endconfiguration %}
 
-## Opening an issue
+### Example: switch configuration
+
+```yaml
+# Example configuration.yaml entry
+modbus:
+  - type: tcp
+    host: IP_ADDRESS
+    port: 502
+    switches:
+      - name: Switch1
+        address: 13
+        write_type: coil
+      - name: Switch2
+        slave: 2
+        address: 14
+        write_type: coil
+        verify:
+      - name: Register1
+        address: 11
+        command_on: 1
+        command_off: 0
+        verify:
+            input_type: holding
+            address: 127
+            state_on: 25
+            state_off: 1
+```
+
+# modbus services
+
+The modbus integration provides two generic write services in addition to the platform-specific services.
+
+| Service | Description |
+| ------- | ----------- |
+| modbus.write_register | Write register or registers |
+| modbus.write_coil | Write coil or coils |
+
+Description:
+
+| Attribute | Description |
+| --------- | ----------- |
+| hub       | Hub name (defaults to 'modbus_hub' when omitted) |
+| unit      | Slave address (0-255), alternative to slave |
```

Comment:
```suggestion
| unit      | Client address (0-255), alternative to client |
```

Reply:
No there are no client address in Modbus, only unit or slave id´s